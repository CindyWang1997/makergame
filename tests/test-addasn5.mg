object main {
  event create
  {
    int x = 5;
    x -= 2.5; // turns to 5 - 2 = 3

    float y = 10.0;
    y -= 3; // turns to 10.0 - 3.0 = 7.0
    std::print::i(x);
    std::print::f(y);
    std::end_game();
  }
}
