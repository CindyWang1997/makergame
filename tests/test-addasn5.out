3
7.000000
